docs/live_update.md

```diff
@@ -30,53 +30,43 @@ The `Tiltfile` at the root of the repo contains this example:
 # tiltdemo1
 k8s_yaml('deployments/demoserver1.yaml')
 dm1_img_name = 'gcr.io/windmill-test-containers/tiltdemo/demoserver1'
-(fast_build(dm1_img_name, 'Dockerfile', '/go/bin/demoserver1')
-  .add('./cmd/demoserver1',
-      '/go/src/github.com/windmilleng/tiltdemo/cmd/demoserver1')
-  .run('go install github.com/windmilleng/tiltdemo/cmd/demoserver1'))
+docker_build(dm1_img_name, '.', dockerfile='Dockerfile.server1')
+live_update(dm1_img_name,
+  [
+    sync('cmd/demoserver1', '/go/src/github.com/windmilleng/tiltdemo/cmd/demoserver1'),
+    run('go install github.com/windmilleng/tiltdemo/cmd/demoserver1'),
+    restart_container(),
+  ])
 ```
 
-This looks similar to the `Tiltfile` in previous tutorials, but instead of building
-with `docker_build()`, it contains `fast_build()`. Let's zoom
-in on that part of the function.
+This looks similar to the `Tiltfile` in previous tutorials, but in addition to building
+with `docker_build()`, it contains `live_update()`. Let's zoom
```

Comment:
either "in addition to building with... it updates with..." or "in addition to calling... it also includes a call to..."

i.e. talk about either the build behavior OR the tiltfile, not both at once.